TheMET/TheMET/Network/ObjectResponse.swift

```diff
@@ -2,14 +2,285 @@
+        self.objectDate = try container.decode(String?.self, forKey: .objectDate)
+        self.objectBeginDate = try container.decode(Int?.self, forKey: .objectBeginDate)
+        self.objectEndDate = try container.decode(Int?.self, forKey: .objectEndDate)
+        self.medium = try container.decode(String?.self, forKey: .medium)
+        self.dimensions = try container.decode(String?.self, forKey: .dimensions)
+        self.measurements = try container.decode(([Measurements]?).self, forKey: .measurements)
+        self.creditLine = try container.decode(String?.self, forKey: .creditLine)
+        self.geographyType = try container.decode(String?.self, forKey: .geographyType)
+        self.city = try container.decode(String?.self, forKey: .city)
+        self.state = try container.decode(String?.self, forKey: .state)
+        self.county = try container.decode(String?.self, forKey: .county)
+        self.country = try container.decode(String?.self, forKey: .country)
+        self.region = try container.decode(String?.self, forKey: .region)
+        self.subregion = try container.decode(String?.self, forKey: .subregion)
+        self.locale = try container.decode(String?.self, forKey: .locale)
+        self.locus = try container.decode(String?.self, forKey: .locus)
+        self.excavation = try container.decode(String?.self, forKey: .excavation)
+        self.river = try container.decode(String?.self, forKey: .river)
+        self.classification = try container.decode(String?.self, forKey: .classification)
+        self.rightsAndReproduction = try container.decode(String?.self, forKey: .rightsAndReproduction)
+        self.linkResource = try container.decode(String?.self, forKey: .linkResource)
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+        let metadataDateString = try container.decode(String.self, forKey: .metadataDate)
+        guard let metadataDate = dateFormatter.date(from: metadataDateString) else {
+            throw DecodeError.dateNotConverted
+        }
+        self.metadataDate = metadataDate
+        
+        self.repository = try container.decode(String?.self, forKey: .repository)
+        self.objectURL = try container.decode(String?.self, forKey: .objectURL)
+        self.tags = try container.decode([Tags]?.self, forKey: .tags)
+        self.objectWikidataURL = try container.decode(String?.self, forKey: .objectWikidataURL)
+        self.isTimelineWork = try container.decode(Bool.self, forKey: .isTimelineWork)
+        self.galleryNumber = try container.decode(String?.self, forKey: .galleryNumber)
+    }
+
+    struct Constituents: Decodable {
+        private let constituentID: Int
+        private let role: String
+        private let name: String
+        private let constituentULAN_URL: String
+        private let constituentWikidata_URL: String
+        private let gender: String
+    }
+
+    struct ElementMeasurements: Decodable {
```

Comment:
It must be `ElementMeasurement`, not plural

Reply:
fixed